src/ng/directive/ngModel.js

```diff
@@ -1337,3 +1403,35 @@ function isObjectEmpty(obj) {
   }
   return true;
 }
+
+
+
+/**
+ * @ngdoc service
+ * @name $modelOptions
+ * @description
+ * This service provides support to the {@link ngModelOptions} directive.
+ *
+ * Here, you can register default options that are used application wide if no {@link ngModelOptions}
+ * directive is found.
```

Comment:
"See ngModelOptions for a list of available options" And should we mention what exactly the default options are?